Bottom: f208998c0e8da6055853819e5704f90e6fd45c3c
Top:    e9de3e95bfce76c034445c699e53f2dbf8ebaaf1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 17:58:19 +0000

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index a06dde1..563bc85 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -342,7 +342,7 @@ pad_callback (const BoxType * b, void *cl)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
